docs/pages/components/select.md

```diff
@@ -411,56 +435,57 @@ The disabled state can also be achieved by adding the `.is-disabled` class or th
 
 
 ## Readonly State
-To make the `Select` component read-only, the readonly attribute needs to be added to the `fd-select__control` element. 
-This can also be done by adding the `.is-readonly` class or the `aria-readonly="true"` attribute.
+To make the `Select` component read-only, the `.is-readonly` class needs to be added to the `fd-select__control` element. 
 
-{% capture disabled-select %}
+{% capture readonly-select %}
 <div class="fd-popover">
-    <div class="fd-popover__control" aria-expanded="false" aria-haspopup="false" aria-readonly="true" readonly>
+    <div class="fd-popover__control">
         <div class="fd-select">
-            <div class="fd-select__control" role="button" aria-expanded="false" aria-haspopup="false" aria-readonly="true" readonly>
-                Select
-           </div>
+            <div class="fd-select__control is-readonly" role="button" aria-expanded="false" aria-label="Select" aria-haspopup="false">Selected Item 2</div>
         </div>
```

Comment:
This is where I still don't know what to do.. I've got the styling added via `is-readonly` class, but how do we communicate it's readonly? The ui5 one has a hidden input. 

Additionally, I'm not sure how about not being able to open the popover -  "Readonly elements are relevant to the user, and application authors SHOULD NOT restrict navigation to the element or its focusable descendants." https://www.w3.org/TR/2017/REC-wai-aria-1.1-20171214/#aria-readonly

Reply:
I've been trying to look for other examples on the internet, and there doesn't appear to be anything like this surprisingly. I agree the language is a little vague on what to do for components with popups. We have a couple steps:

1. Like UI5, we can add a hidden `input` with `aria-readonly` as a child of the button. This is so screen readers can tell it is read-only when focused without have to click the button.

1. (optional?) Allow the popover to open in read only mode, and set `aria-readonly` on the `ul` `listbox`.

We should definitely do 1. 2 is potentially optional. The disappearance of the chevron in readonly suggests that it shouldn't be opened, and I can't find any mandate where the list of options needs to be presented in read-only mode.

Reply:
I think it's ok to say that a combobox is `aria-readonly` only when it's value can't be set. Technically you can't type in the box, but you can change its value. The definition in the link Jenna provided for `aria-readonly` says `The user can set the value of the element.` vs `The user cannot change the value of the element.` By that I don't think it's limited to actually typing only, but any sort of change in value whether it be selecting an option from a dropdown or a radio group, setting a checkbox, etc.

Reply:
@meganaconley @jacobdevera Stupid question - but I'm not following 😆  What should the html look like?

Reply:
@jbadan Yeah sorry, I was thinking out loud and got kinda convoluted 😅

Like UI5 (https://sapui5.hana.ondemand.com/#/entity/sap.m.Select/sample/sap.m.sample.Select), I would try to change to `combobox` instead of `button` because it supports `aria-readonly` :
`<div class="fd-select__control is-readonly" role="combobox" aria-expanded="false" aria-label="Select" aria-haspopup="false" aria-readonly="true">Selected Item 2</div>`

Reply:
Thank you!

Reply:
It looks like if we go `role="combobox"` we need a bunch of other changes, no? For example, including an input that's always aria-readonly (not just for this example)

That leads me to, should this div just be an input with `aria-readonly='true' role='combobox'`?

Reply:
I don't think the hidden input in the ui5 example is actually contributing anything. Once the role is set to `combobox`, it shouldn't matter if it's a `div` or `input` other than if it's easier for styling purposes.

Reply:
So this is what I was running into: https://www.w3.org/TR/wai-aria-1.1/#combobox

"Authors MUST ensure an element with role combobox contains or owns a text input element with role textbox or searchbox and that the text input has aria-multiline set to false. "

Reply:
Technically, we are already breaking spec because we are not supplying an editable textbox input for the combobox in the first place; we're merely using it as a way to get `aria-readonly` to be supported. The spec mentions that the `input` must be a child of the combobox, not the combobox itself. In our case, if we supply the fake hidden input element like UI5 does, at best, it doesn't affect anything for screen reader callouts, and at worst, could be misleading for when the Select is not marked readonly.

Again, whether the base combobox is an `input` or `div` doesn't affect accessibility because the semantics are being overriden by the combobox role.

My take: I'm okay with adding a hidden input into a combobox like the one in my code snippet above, but only for the readonly state. All other select states should remain with the usual listbox in `button` as in the w3c example.

Reply:
But what I'm trying to say - should we even be using `role="combobox"` if we're not following the spec? 

This is starting to feel like too hacky of a workaround. We can also just table this and put an actual story in the backlog to investigate this further since we've spent so much time on it already.

Reply:
Ideally, no. Agreed that it can wait for a future story.

Reply:
UIA11Y-2324